src/modules/auth/auth.service.ts

```diff
@@ -18,15 +18,20 @@ export class AuthService {
   ) {}
 
   async login(loginDto: LoginDto): Promise<{ token: string }> {
-    const user = await this.userRepository.findOneBy({ email: loginDto.email });
-    if(!user){
-      throw new Error('User not found')
+    try {
+      const user = await this.userRepository.findOneBy({ email: loginDto.email });
+      if (!user) {
+        throw new Error('User not found');
```

Comment:
throw new BadRequestException({
        message: ERRORS_DICTIONARY. USER_NOT_FOUND
});
You had better only modify the error-dictionary.constraint.ts file in the constraints folder. Avoid hardcoding.